### PR TITLE
[Ready] Gives Synthetic lizards a existing blood type and blue!

### PR DESCRIPTION
**Pull request by Strazyplus**
## UPDATE: COMPROMISING BLUE
[ APPARENTLY SYNTHETIC BLOOD IS OP OR SOME SHIT](https://www.youtube.com/watch?v=68ugkg9RePc)
image
[whatever solution: switch to IPC blood but fuck that black shit 
**I MADE IPC BLOOD **BLUE**** YOUR WELCOME](https://www.youtube.com/watch?v=68ugkg9RePc)

## RELEASED BLUE
[Synthlizards had a bug in their snowflake code, TLDR a simple misspelling reference to Synthetic blood was incorrect which caused synths blood to be red as if they are organic like.... HUMANS
no no no no..
their blood is **blue**!](https://www.youtube.com/watch?v=68ugkg9RePc)

[SHOW THE MAGIC! (KINDA) I made it modular](https://www.youtube.com/watch?v=68ugkg9RePc)
ALp34M

image
IMAGINE THE S GETS TURNED INTO A HF INSTEAD.

## Why It's Good For The Game

[**SYNTHETIC LIZARDS ARE SYNTHETIC AND SYNTHETIC BLOOD IS BLUE BUT SINCE SOME PEOPLE COMPLAINED THAT SYNTHETIC BLOOD IS BULLSHIT OP, WHATEVER I MADE SYNTHETIC LIZARDS BLOOD TO IPC BLOOD BUT THAT BLACK BLOOD IS UNCOOL, SO I MADE IPC BLOOD BLUE**](https://www.youtube.com/watch?v=68ugkg9RePc)

## Changelog
:cl:
tweak: tweaked synthetic lizard exotic blood type to HF (IPC blo & made IPC blood **blue**
fix: fixed synthetic lizard exotic blood type to a functioning blood type.
/:cl:

### CODE REVIEW

**Comment by LimblessLayabout** [comment]
So long as I can give synthlizards chem/synthetic_blood without poisoning them to death, I'm  for it.

**Comment by Strazyplus** [comment]
> So long as I can give synth lizards chem/synthetic_blood without poisoning them to death, I'm for it.

This fixes that likely because their blood type was beyond exotic, synth lizards could only use other synth lizard blood. how and why? because of the typo in their bloodtype.

**Comment by Varoxus** [comment]
Actually Synth lizards can't use other synth lizard blood or even their own blood currently, when injected or infused back into themselves it damages them presumably because 'S' type blood isn't really a thing. /shrug

File: code/modules/mob/mob_helpers.dm [hunk 1]
```diff
@@ -507,7 +507,7 @@ It's fairly easy to fix if dealing with single letters but not so much with comp
 /mob/proc/can_hear()
 	. = TRUE
 
-/proc/bloodtype_to_color(var/type)
+///proc/bloodtype_to_color(var/type) // skyrat change moved to modular skyrat
 	. = BLOOD_COLOR_HUMAN
 	switch(type)
 		if("U")//Universal blood; a bit orange
```
Comment (Useroth) [inline code/modules/mob/mob_helpers.dm:513]: You should comment that entire proc out, not just the declaration line, if you're moving it.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -74,7 +74,7 @@
 		B.reagents.add_reagent(type, reac_volume)
 	B.update_icon()
 
-/datum/reagent/blood/on_new(list/data)
+//datum/reagent/blood/on_new(list/data) // Skyrat change moved to modular
 	if(istype(data))
 		SetViruses(src, data)
```
Comment (Useroth) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:79]: Same thing here.

**Comment by Strazyplus** [comment]
Another PR showed up and did it but blood is not blue.


RIP

**Pull request closed without merging**